### ISSUE TITLE
Help Wanted: Screenshot doesn't represent user's product experience

### ISSUE THREAD

**Issue by cwarnermm** [mattermost/docs#7067]
# Issue
The first screenshot in [Keyboard accessibility - Mattermost documentation](https://docs.mattermost.com/collaborate/keyboard-accessibility.html) is missing the blue focus outline indicating that the area is selected/active). The screenshot is also outdated and missing a number of product enhancements.

## Current
![image](https://github.com/mattermost/docs/assets/74422101/6e9e0bb7-5989-4b6d-8d5e-a93bf9e8cd16)

## Expected

![image](https://github.com/mattermost/docs/assets/74422101/dd00622e-6c9f-4050-9a06-219bf5387eed)

# How to Contribute
- Re-capture the UI, with accessibility shortcuts enabled and blue outline visible.
- Any public channel on the [Mattermost Community Server](https://community.mattermost.com) can be captured with an empty text box (no message draft).
- Capture using a browser or the desktop app displaying the **Mattermost Denim Theme**.

**Comment by Rutam21** [comment]
@cwarnermm I would like to work on this. Feel free to assign this to me. Thanks.
